### DISCUSSION TITLE
Delete Record throwing error on weak supervision run

### DISCUSSION THREAD

**Discussion by lakshmid13579** [code-kern-ai/refinery#181] [General]
I deleted a record and tried to run active learning with  weak supervision on the labeling function heuristic. I'm getting the below error as it is expecting deleted record. Can you please let me know if there is any solution to it?
![image](https://user-images.githubusercontent.com/25331738/202253697-a8ba9065-e0a2-4171-a716-051fe3d4c55d.png)

**Comment by jhoetter** [comment]
Hi @LakshmiPanguluri, I just openend an issue (#182) for this. We're working on it :)
